### PR TITLE
Update SelectListGeneratorVTwo.java

### PR DESCRIPTION
**Pull request by sachaj**
For : 
./api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java

Changed :
ln23-24 : tab to spaces and added empty line
lns56-156 : tabs to spaces

### CODE REVIEW

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java [hunk 1]
```diff
@@ -54,103 +55,103 @@
...
+        for (String key:hmap.keySet()) {
+            String[] x = new String[2];
+            x[0] = key;
+            x[1] = hmap.get(key);
+            objectsToSort.add(x);
+        }
+
+        //if no comparator is passed in, utilize MapPairsComparator
+        if(comparator == null) {
+            comparator = new MapPairsComparator(vreq);
```
Comment (awoods) [inline api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java:116]: I changed the target of this pull-request from `master` to `sprint-i18n-whitespace`. Nothing additional needs to be done for that.

I then locally pulled down this pull-request to check the corrections with the following command:
```
git diff pull/150 master api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java
```
The result of that command shows that your updates are perfect... except in line:116 (here), and lines 142-146 below.

Screenshot from 2020-04-02 10-30-59

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java [hunk 2]
```diff
@@ -54,103 +55,103 @@
...
+        public MapPairsComparator(VitroRequest vreq) {
+            this.collator = vreq.getCollator();
+        }
+        public int compare (String[] s1, String[] s2) {
+            if (s2 == null) {
+                return 1;
+            } else if (s1 == null) {
+                return -1;
+            } else {
+                if ("".equals(s1[0])) {
```
Comment (awoods) [inline api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/fields/SelectListGeneratorVTwo.java:142]: See comment above.

**Comment by awoods** [comment]
@sachaj : it also appears that you need to update your branch based on recent updates to this file.

**Comment by awoods** [comment]
I am still seeing a difference between `master` and this pull-request on the lines updated in the latest commit.
I would suggest starting from a copy of the SelectListGeneratorVTwo.java found in `master`, then simply paste the block added by Michel:
https://github.com/awoods/Vitro/pull/1/files?diff=unified&w=1#diff-06fe26f352b5893646862bdd4befd23fR56-R96

**Comment by sachaj** [comment]
ok, I've started over and now am seeing the master and the corrected whitespace version with Michel's additions ln57-97, my version seems to be up to date with all corrections. Should i submit this file now anyway?

**Comment by awoods** [comment]
I you have an updated file... please add a commit to this pull-request.
Note, since there is a merge conflict, you should update your "my-whitespace" branch first:
```
$ git checkout my-whitespace
$ git pull
```
..resolve conflicts

**Comment by sachaj** [comment]
Sorry Andrew, I'm a bit lost at this point. My file is now not updated, it's a fresh version. I've reset everything and now am getting this : 
$ git commit -m "updating files, adding commit"
On branch my-whitespace
Your branch is up to date with 'origin/my-whitespace'.
nothing to commit, working tree clean

**Comment by awoods** [comment]
Superseded by: https://github.com/vivo-project/Vitro/commit/6fbaab82c1e53d3015ed15866d4ce4dd744dfa0d#diff-06fe26f352b5893646862bdd4befd23f

**Pull request closed without merging**